### PR TITLE
increased TRDM logging and removed deprecated func

### PR DESCRIPTION
**Pull request by cameroncaci**
Added more logging for TRDM as well as began pruning out response body print outs. Additionally removed deprecated func.

### CODE REVIEW

File: pkg/trdm/get_table.go [hunk 1]
```diff
@@ -6,43 +6,51 @@ import (
...
 	if err != nil {
+		logger.Error("failed to read response body", zap.Error(err))
 		return err
 	}
 	defer resp.Body.Close()
 
 	// Parse it into getTableResponse model
 	err = json.Unmarshal(body, &getTableResponse)
 	if err != nil {
+		logger.Error("faield to unmarshal response body into getTableResponse type", zap.Error(err))
```
Comment (danieljordan-caci) [inline pkg/trdm/get_table.go:41]: typo with the word `faield` ya goober

File: pkg/trdm/get_table.go [hunk 2]
```diff
@@ -6,43 +6,51 @@ import (
...
 		return err
 	}
 
 	// Parse the attachment, this will also store it in the DB if all goes well
 	err = parseGetTableResponse(appCtx, getTableResponse.Attachment, getTableRequest.PhysicalName)
 	if err != nil {
+		logger.Error("faield to parseGetTableResponse and store it into the database", zap.Error(err))
```
Comment (danieljordan-caci) [inline pkg/trdm/get_table.go:48]: same typo here

File: pkg/trdm/get_table.go (hunk 1 above)
Comment (cameroncaci) [reply pkg/trdm/get_table.go]: I had faield to see that, fixed now!

**Review by danieljordan-caci — APPROVED** [review]
WINNER

**Review by traskowskycaci — APPROVED** [review]

**Review by TevinAdams — APPROVED** [review]

**Pull request merged**